plugins/plugin-chart-pivot-table/package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@superset-ui/chart-controls": "0.17.22",
     "@superset-ui/core": "0.17.22",
-    "@kgabryje/react-pivottable": "^0.12.2"
+    "@kgabryje/react-pivottable": "^0.12.3"
```

Comment:
@kgabryje  Can we move this under `@superset-ui` ? Just invited you on both GitHub and NPM. You should have access to fork this repo into `apache-superset` and publish npm packages under `@superset-ui`. If not, let me know!

Reply:
Done 👍